README.md

```diff
@@ -1,4 +1,27 @@
-# Gateway API Inference Extension 
+# Gateway API Inference Extension (GIE)
+
+This project offers tools for [AI Inference], enabling developers to build [Inference Gateways].
+
+[AI Inference]:https://www.digitalocean.com/community/tutorials/llm-inference-optimization
+[Inference Gateways]:#concepts-and-definitions
+
+## Concepts and Definitions
+
+AI/ML is changing rapidly, and [Inference] goes beyond basic networking to include complex traffic routing and optimizations. Below are key terms for developers:
+
+- **Scheduler**: Makes decisions about which endpoint is optimal (best cost / best performance) for an inference request based on `Metrics and Capabilities` from [Model Serving Platforms].
```

Comment:
We have been using the term "endpoint picker" in pkg and image names in this repo. I think we need to somehow introduce it here. For example 


Perhaps:

```suggestion
- **Scheduler**: Makes decisions about which endpoint is optimal (best cost / best performance) for an inference request based on `Metrics and Capabilities` from [Model Serving Platforms]. We refer to this scheduler as the [`Endpoint Picker`](https://github.com/kubernetes-sigs/gateway-api-inference-extension/tree/main/pkg/epp).
```

Reply:
I guess EPP as it is right now is the full extension that implements both the scheduler and the metrics+capabilities components, not just the scheduler. Perhaps we add it as a separate definition after the definitions of scheduler and metrics+capabilities.

Reply:
Sounds good, I added it as a new line, LMKWYT?